### ISSUE TITLE
window.webkitStorageInfo is deprecated on Chrome.

### ISSUE THREAD

**Issue by timcombs** [intercellular/cell#122]
I'm playing with cell.js on Chrome and I get this error message in the Developer Console:

[Deprecation] 'window.webkitStorageInfo' is deprecated. Please use 'navigator.webkitTemporaryStorage' or 'navigator.webkitPersistentStorage' instead.
(anonymous) @ cell.js:348
detect @ cell.js:346
create @ cell.js:371
(anonymous) @ cell.js:395

**Comment by devsnek** [comment]
First of all that isn't an error. This happens because Cell iterates through `Object.keys` of `window`

Chrome has setup a [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) on `window.webkitStorageInfo` to show a console warning telling developers that it is deprecated so when Cell tries to see it `window.webkidStorageInfo.$cell` is a thing that deprecation warning will show.

TLDR; nothing to worry about.
